src/lib/features/project/project-service.ts

```diff
@@ -305,16 +305,16 @@ export default class ProjectService {
         }
     }
     async generateProjectId(name: string): Promise<string> {
-        const generateUniqueId = async (name: string, suffix?: number) => {
-            const slug = createSlug(name);
+        const slug = createSlug(name).slice(0, 90);
+        const generateUniqueId = async (suffix?: number) => {
```

Comment:
We don't need to recalculate the slug every time - it'll never change -  so I moved the variable assignment out, which also means we can remove that argument from the inner function.